mozdownload/scraper.py

```diff
@@ -326,20 +305,15 @@ def total_seconds(td):
 
                 if log_level <= logging.INFO and content_length:
                     pbar.finish()
-                break
-            except (requests.exceptions.RequestException, errors.TimeoutError), e:
-                if tmp_file and os.path.isfile(tmp_file):
+            except:
+                if os.path.isfile(tmp_file):
```

Comment:
can't be: https://github.com/mozilla/mozdownload/pull/342/files#diff-c9f02456f02d4f3782e1a0f09a60898cL288